package.json

```diff
@@ -174,7 +178,8 @@
   },
   "resolutions": {
     "**/websocket-extensions": "^0.1.4",
-    "**/graphql-playground-middleware-express": "^1.7.18"
+    "**/graphql-playground-middleware-express": "^1.7.18",
+    "@emotion/core": "10.0.28"
```

Comment:
Storybook depends on an older version, and was being used in the component build flow causing issues.

Reply:
We'll need to make sure to update this if we update styling-library.